spec/models/tool_renting_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe ToolRenting, type: :model do
+  describe :Validations do
+    it 'check presence renter' do
+      tool_renting = build(:tool_renting)
```

Comment:
These can be optimized a bit.

`build(:tool_renting)` will in fact create all associations, so every time you call it, 
a tool will be created and saved to database. (I did not know this before)

You can however test validations by doing:
`ToolRenting.new.should validate_presence_of(:tool)`